### ISSUE TITLE
Cannot Launch Simulation

### ISSUE THREAD

**Issue by yetongumich** [duckietown-udem/udem-fall19-public#8]
Hi, I've encountered a problem when trying to setup the simulation environment by following `notebooks/01-duckietown-classical-baseline.ipynb`. Everything went on well until I tried to run `roslaunch notebooks/lf_slim.launch`, it does not find a launch file named `inverse_kinematics_node.launch`.
![image](https://user-images.githubusercontent.com/22564146/66541537-99d4db00-eafd-11e9-9171-85f44c0a2254.png)

**Comment by yetongumich** [comment]
I was following the instructions in `notebooks/01-duckietown-classical-baseline.ipynb`.
Note that I changed `docker-compose -f docker-compose-lf.yml build` to `docker-compose -f docker-compose.yml build` because there is no file `docker-compose-lf.yml` in the directory.

**Comment by liampaull** [comment]
inverse kinematics and kinematics don't do the same thing. Try the second hw exercise for now - I will update the first one

**Comment by yetongumich** [comment]
Sounds good!
Here is a list of issues I encountered when following the first notebook:
1. When trying to clone the repo with the provided command `git clone https://github.com/duckietown/udem-fall19-public --recursive`, the old link seems expired. I instead clone this repo.
2. There is no docker-compose-lf.yml in the repository, so I switched to docker-compose.yml
3. Directly running `roslaunch notebooks/lf_slim.launch` results in some errors, and I changed line 70 with `kinematics_node.launch`
4. After all these, the visualization interface does not show up.
